react-backend/client/src/components/visitors/qrcode.js

```diff
@@ -161,6 +120,48 @@ export class QRCode extends Component {
     }
   }
 
+  handleVideo = () => {
+    const newState = {};
```

Comment:
Is this overkill, can it be done by:
```
this.setState({login: this.state.login + 1})
```

Reply:
Yes. Can also add this, but again, just wanted to make the min changes.